.github/workflows/test_fast.yml

```diff
@@ -21,11 +21,23 @@ jobs:
       matrix:
         os: ['ubuntu-latest']
         python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3']
+        time-zone: ['UTC']
         include:
+          # mac os test
           - os: 'macos-latest'
-            python-version: '3.7'
+            python-version: '3.7'  # oldest supported version
+            time-zone: ['UTC']
+
+          # non-utc timezone test
+          - os: 'ubuntu-latest'
+            python-version: '3.9'  # not the oldest, not the most recent version
+            time-zone: ['XXX-09:35']
```

Comment:
It might not be happy with 'UTC' (the format for the `TZ` env var is kinda weird).

Maybe try blank for UTC, in isodatetime it is set up like this:

```diff
diff --git a/.github/workflows/test_fast.yml b/.github/workflows/test_fast.yml
index 2f303e36e..ae015b045 100644
--- a/.github/workflows/test_fast.yml
+++ b/.github/workflows/test_fast.yml
@@ -21,9 +21,15 @@ jobs:
       matrix:
         os: ['ubuntu-latest']
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3']
+        python-version: ['3.7', '3.8', '3.10', '3.11', '3']
+        include:
+          - os: 'ubuntu-latest'
+            python-version: '3.9' # not the oldest nor most recent version
+            tz: 'XXX-09:35' # UTC+09:35
         include:
           - os: 'macos-latest'
             python-version: '3.7'
+    name: ${{ matrix.os }} py-${{ matrix.python-version }} ${{ matrix.tz }}
     env:
+      TZ: ${{ matrix.tz }}
       PYTEST_ADDOPTS: --cov --cov-append -n 5 --color=yes
     steps:
```

Reply:
Ok, switched to blank.